guides/common/modules/proc_performing-a-full-backup.adoc

```diff
@@ -43,6 +43,27 @@ You can use the `{foreman-maintain} backup` command to back up remote databases.
 
 You can use both online and offline methods to back up remote databases, but if you use offline method, the `{foreman-maintain} backup` command performs a database dump.
 
+.Backing up to a remote NFS share
+
+You can back up your {ProjectServer} or {SmartProxyServer} in a remote NFS share.
+
+* Make sure you export the share in your NFS server with the `no_root_squash` option to disable root squashing.
```

Comment:
I think that is bad advise, even if we add that warning at the end. How about:
```suggestion
* Make sure the `root` user of your {ProjectServer} or {SmartProxyServer} can write to the NFS share.
NFS export options like `root_squash` and `all_squash` are known to prevent this.
For more information, see https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/configuring_and_using_network_file_services/deploying-an-nfs-server_configuring-and-using-network-file-services
```

Especially, you can backup to `root_squash`ed NFS share just fine if the directory you write to is writable by `nobody` on the NFS server (and you use `--preserve-directory` to foreman-maintain, but that part is a bug in f-m that I am going to fix).

Reply:
https://github.com/theforeman/foreman_maintain/pull/888 for the code change

Reply:
@asteflova I would say, 
* Have QE do some testing with some recommended options
* The test should be done for both backup as well as restore
* Once done and confirmed to be successfully happening, Document those options as primary recommendations

An end-user can export the NFS share with different combinations of options. So the best we can do is to provide:
* Minimum recommended options that would work
* Some NFS export options that may affect the functionality and should not be used

Reply:
Well, to be honest, my whole point is that we should NOT document ANY NFS options as we're not managing the NFS server. Especially as those options vary in meaning depending on the NFS implementation.

E.g. I have a NetApp NFS here, which I assume has `root_squash` on, as any file I create as `root` gets `nobody` as owner instead, but I can backup there just fine as `chown .postgres` works. When I replicate the setup with an EL9 NFS server, I get "Operation not permitted" for the `chown`.

Reply:
Without any recommendations at all, The end-users run into permission errors and come back to RH for further support on resolving those matters. So if we mention something like NFS share can be used, we also should mention how they can confirm that the NFS share is usable.

So if not recommendations about exporting NFS share, We can document some points about how to check\test whether the exported share is usable for a backup\restore operation or not.

Reply:
> So if we mention something like NFS share can be used, we also should mention how they can confirm that the NFS share is usable.

That's why I wrote "Make sure the `root` user of your can write to the NFS share." as that's exactly the test for usability of the share. Which btw also applies to any other storage, not only NFS (it's just for local permission checks `root` usually has `DAC_OVERRIDE` capability and can ignore any constraints).

Reply:
Applied.